From 72e084165e1e935e2ddf030481f18a77ea5b3cc2 Mon Sep 17 00:00:00 2001
From: Topi Reinio <topi.reinio@theqtcompany.com>
Date: Thu, 26 May 2016 10:10:32 +0200
Subject: [PATCH] Doc: Fix documentation warnings for
 QSslDiffieHellmanParameters

qssldiffiehellmanparameters.cpp:107: warning: Unknown command '\encoded'
qssldiffiehellmanparameters.cpp:131: warning: Unknown command '\device'
qssldiffiehellmanparameters.cpp:276: warning: Cannot find 'operator!=(...)'
in '\fn' bool QSslDiffieHellmanParameters::operator!=(const QSslDiffieHellmanParameters &other) const

Change-Id: Ieb0f27f0c01c799323f4b69b7f2474513c67d443
Reviewed-by: Venugopal Shivashankar <Venugopal.Shivashankar@qt.io>
Reviewed-by: Timur Pocheptsov <timur.pocheptsov@theqtcompany.com>
---
 src/network/ssl/qssldiffiehellmanparameters.cpp | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/src/network/ssl/qssldiffiehellmanparameters.cpp b/src/network/ssl/qssldiffiehellmanparameters.cpp
index 7442411..d411ea0 100644
--- a/src/network/ssl/qssldiffiehellmanparameters.cpp
+++ b/src/network/ssl/qssldiffiehellmanparameters.cpp
@@ -104,7 +104,7 @@ QSslDiffieHellmanParameters::QSslDiffieHellmanParameters()
 
 /*!
     Constructs a QSslDiffieHellmanParameters object using
-    the byte array \encoded in either PEM or DER form.
+    the byte array \a encoded in either PEM or DER form.
 
     After construction, the isValid() method should be used to
     check whether the Diffie-Hellman parameters were valid and
@@ -128,7 +128,7 @@ QSslDiffieHellmanParameters::QSslDiffieHellmanParameters(const QByteArray &encod
 
 /*!
     Constructs a QSslDiffieHellmanParameters object by
-    reading from \device in either PEM or DER form.
+    reading from \a device in either PEM or DER form.
 
     After construction, the isValid() method should be used
     to check whether the Diffie-Hellman parameters were valid
@@ -273,12 +273,6 @@ bool operator==(const QSslDiffieHellmanParameters &lhs, const QSslDiffieHellmanP
     return lhs.d->derData == rhs.d->derData;
 }
 
-/*! \fn bool QSslDiffieHellmanParameters::operator!=(const QSslDiffieHellmanParameters &other) const
-
-  Returns \c true if this QSslDiffieHellmanParameters is not equal to \a other; otherwise
-  returns \c false.
-*/
-
 #ifndef QT_NO_DEBUG_STREAM
 /*!
     \relates QSslDiffieHellmanParameters
